build/build.proj

```diff
@@ -121,6 +121,36 @@
              />
   </Target>
 
+  <!-- Restore SDKs -->
+  <Target Name="DeployMSBuildSdks" AfterTargets="BuildNuGetPackages;RebuildNuGetPackages">
+    <ReadLinesFromFile File="$(RepositoryRootDirectory)TestAssets\buildVersion.txt">
+      <Output TaskParameter="Lines" ItemName="SdkBuildVersion" />
+    </ReadLinesFromFile>
+    
+    <PropertyGroup>
+      <SdkBuildVersion>@(SdkBuildVersion)</SdkBuildVersion>
+      <_RestoreSdkCommand>$(DotNetTool) restore</_RestoreSdkCommand>
+      <_RestoreSdkCommand>$(_RestoreSdkCommand) $(RepositoryRootDirectory)\build\Sdk\SdkRestoreProject.proj</_RestoreSdkCommand>
+      <_RestoreSdkCommand>$(_RestoreSdkCommand) /p:ExcludeRestorePackageImports=true</_RestoreSdkCommand>
+      <_RestoreSdkCommand>$(_RestoreSdkCommand) --packages $(NuGet_Packages)</_RestoreSdkCommand>
+    </PropertyGroup>
+    
+    <Exec Command="$(_RestoreSdkCommand)" />
+
+    <ItemGroup>
+      <SDKFile Include="$(PackagesLayoutDir)\**\*.*">
+        <SDKFolder>microsoft.net.sdk</SDKFolder>
```

Comment:
Or SdkDir since that abbreviation is pervasive in msbuild: OutDir and this is alongside RecursiveDir.